modules/programs/yambar.nix

```
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.yambar;
+  yamlFormat = pkgs.formats.yaml { };
+
+in {
+  options = {
+    programs.yambar = {
+      enable = mkEnableOption "Yambar";
+
+      package = mkPackageOption pkgs "yambar" { };
+
+      settings = mkOption {
+        type = yamlFormat.type;
+        default = { };
+        example = literalExpression ''
+          bar = {
+            location = "top";
+            height = 26;
+            background = "00000066";
+
+            right = [
+              {
+                clock.content = [
+                  {
+                    string.text = "{time}";
+                  }
+                ];
+              }
+            ];
+          };
+        '';
+        description = ''
+          Configuration written to
+          <filename>$XDG_CONFIG_HOME/yambar/config.yml</filename>.
+          See
+          <citerefentry>
+           <refentrytitle>yambar</refentrytitle>
+           <manvolnum>5</manvolnum>
+          </citerefentry>
+          for options.
```

Can switch to Markdown

```suggestion
          Configuration written to
          {file}`$XDG_CONFIG_HOME/yambar/config.yml`.
          See {manpage}`yambar(5)` for options.
```